site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -240,7 +259,8 @@ Content scripts can be injected as files…
 {% Label %}content-script.js:{% endLabel %}
 
 ```js
-document.body.style.backgroundColor = 'orange';
+//// content-script.js ////
```

Comment:
@jpmedley what's the current practice for identifying the filename associated with a code snippet?

Reply:
We can agree on this later.

Reply:
We've agreed to:

- Remove the label for now (It [exists](https://developer.chrome.com/docs/handbook/components/#labels), but @jpmedley has concerns about the formatting until we tweak the CSS)
- Open an issue to discuss how to identify filenames

Reply:
As discussed, I've removed the label for now and opened https://github.com/GoogleChrome/developer.chrome.com/issues/4948 to discuss further.